src/components/common/copyable/copy-to-clipboard-button/copy-to-clipboard-button.test.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * SPDX-FileCopyrightText: 2022 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+/*
+ * SPDX-FileCopyrightText: 2022 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import { mockI18n } from '../../../markdown-renderer/test-utils/mock-i18n'
+import { CopyToClipboardButton } from './copy-to-clipboard-button'
+import { render, screen } from '@testing-library/react'
+import * as uuidModule from 'uuid'
+
+jest.mock('uuid')
+
+describe('Copy to clipboard button', () => {
+  const copyContent = 'Copy McCopy Content. Electric Copyloo'
+  const copyToClipboardButton = <CopyToClipboardButton content={copyContent} />
+  const uuidMock = 'https://xkcd.com/221/'
```

Comment:
Although this seems a funny reference to randomness of UUIDs, I'd rather have some real UUID in here to ensure that characters like the colon or slashes don't have sideeffects (now or in the future).